### ISSUE TITLE
Recent migration of deployment process where a windows service step is incorrectly saved with a package array

### ISSUE THREAD

**Issue by jburger** [OctopusDeploy/Issues#5191]
Malformed 'Deploy a windows service' steps, regardless of whether or not they are disabled, can cause the deployment process screen to throw a client side error. 

These steps would have to have been malformed as a result of [this bug](https://github.com/OctopusDeploy/Issues/issues/4876) or created in such a way that the package ID was removed prior to an upgrade to `2018.9.x`

## What I expected to happen

Deploy a Windows service steps should render correctly at all times.

## Steps to reproduce

1. Create a deployment on a version prior to `2018.9.x` using the 'deploy a windows service' step - leveraging this bug https://github.com/OctopusDeploy/Issues/issues/4876 to remove the package ID
2. Upgrade instance to `2018.9.x` or greater
3. Navigate to the deployment process
4. Observe error (see log excerpt below)

### Log excerpt
Errors generally appear in the browser when accessing the Deployment process screen 
```
Octopus v2018.9.16

TypeError: Cannot read property ‘PackageId’ of undefined

render (/app/components/Actions/windowsService/windowsServiceAction.tsx:30:62)
Lf (/node_modules/react-dom/cjs/react-dom.production.min.js:147:337)
sf:a:{e=b.type;c=b.pendingProps;d=b.memoizedProps;if(nf())null===c&& (/node_modules/react-dom/cjs/react-dom.production.min.js:150:232)
e (/node_modules/react-dom/cjs/react-dom.production.min.js:182:349)
g (/node_modules/react-dom/cjs/react-dom.production.min.js:183:347)
p (/node_modules/react-dom/cjs/react-dom.production.min.js:184:366)
0;null!==a&&c;){c=!1;if(a.pendingWorkPriority===T||a.pendingWorkPriority>b)c=!0,a.pendingWorkPriority=b;null!==a.alternate&&(a.alternate.pendingWorkPriority===T||a.alternate.pendingWorkPriority>b)&&(c=!0,a.alternate.pendingWorkPriority=b);if(null===a[“return”])if (/node_modules/react-dom/cjs/react-dom.production.min.js:188:389)
S (/node_modules/react-dom/cjs/react-dom.production.min.js:187:415)
enqueueSetState (/node_modules/react-dom/cjs/react-dom.production.min.js:140:181)
isMounted (/node_modules/react/cjs/react.production.min.js:12:343)
```

## Affected versions

**Octopus Server: version 2018.9.x and above** 

## Workarounds

It is possible to use the REST API (e.g. via octopus.client library calls) to update the JSON manually, or simply remove the problematic steps entirely and then recreate the step. 

Example: https://octopus-help.s3.dualstack.us-west-2.amazonaws.com/original/3X/b/b/bb3b23216f5d2fd1d294cc162804a848609f4843.linq

## Links

Related to: https://github.com/OctopusDeploy/Issues/issues/4876

[Public report](https://help.octopus.com/t/an-unexpected-error-occurred-in-octopus-v2018-9-16-typeerror-cannot-read-property-packageid-of-undefined/22006)

**Comment by distantcam** [comment]
Had a customer hit this issue with lots of broken project. Modified the fix script to go through all processes and find broken steps, and set their package to `BROKEN-REPLACEME` with feed `feedid`.

[fix-issue-22006-for-customer.zip](https://github.com/OctopusDeploy/Issues/files/2743648/fix-issue-22006-for-customer.zip)

**Comment by hnrkndrssn** [comment]
Another report https://help.octopus.com/t/an-unexpected-error-occurred-in-octopus-v2018-10-5/22728

**Comment by Dalmirog** [comment]
Another report (+ related slack conversation) https://octopusdeploy.slack.com/archives/CG4JP65N2/p1562809671320300

**Comment by tmbam** [comment]
One more report.
https://secure.helpscout.net/conversation/932991927/48894?folderId=2845830

**Comment by tothegills** [comment]
Looks like this was fixed in 2018.11.0 by: https://github.com/OctopusDeploy/OctopusDeploy/commit/524f82a1d838c403f70e11e3845390502128fc25
